includes/settings/settings-ppec.php

```diff
@@ -400,4 +457,168 @@
 			'drop' => __( 'Do not send line items to PayPal', 'woocommerce-gateway-paypal-express-checkout' ),
 		),
 	),
-) );
+
+	'button_settings' => array(
+		'title'       => __( 'Button Settings', 'woocommerce-gateway-paypal-express-checkout' ),
+		'type'        => 'title',
+		'description' => __( 'Customize the appearance of Express Checkout on your site.', 'woocommerce-gateway-paypal-express-checkout' ),
+	),
+	'use_spb' => array(
+		'title'       => __( 'Smart Payment Buttons', 'woocommerce-gateway-paypal-express-checkout' ),
```

Comment:
Duplicate key.

Reply:
Thanks – not sure when or how the latter one got changed from `label` to `title`.